src/aspire/basis/fle_2d.py

```diff
@@ -653,22 +673,22 @@ def lowpass(self, coeffs, bandlimit):
         :param bandlimit: Integer bandlimit (max frequency).
         :return: Band-limited coefficient array.
         """
-        if len(coeffs.shape) == 1:
-            coeffs = coeffs.reshape((1, coeffs.shape[0]))
-        assert (
-            len(coeffs.shape) == 2
-        ), "Input a stack of coefficients of dimension (num_images, self.count)."
-        assert (
-            coeffs.shape[1] == self.count
-        ), "Number of coefficients must match self.count."
+
+        if not isinstance(coeffs, Coef):
```

Comment:
Changed.

I guess the documentation was fine in Chris's review.

Reply:
Guess we all missed the blank line on Chris's review. 

FWIW, I only pointed out the instances of this param change where Numpy was explicitly mentioned.

Reply:
Sure. Next time make an issue and assign it to yourself if it is not relevant to the code under review.

Reply:
Eh, I thought it was relevant. We changed the acceptable input type which made the param description incorrect. If you're talking about the mention of the blank line, there's already an issue about docstring linting.

Reply:
For sure, the type change was relevant. 

I'm glad you found your assignment from last year addressing the other doc concern you brought up. I'll send you an email about my thoughts on it since this isn't really the forum. Thanks